app/models/post.rb

```diff
@@ -0,0 +1,18 @@
+class Post < ApplicationRecord
+  has_many :likes
+  has_many :comments
+
+  belongs_to :author, class_name: 'User'
+
+  after_save :update_post_user_counter
+
+  private
+
+  def post_counter
+    author.increment!(:posts_counter)
+  end
+
+  def recent_comments
+    comments.limit(5).oder(created_at: :desc)
+  end
```

Comment:
This method should come before line 9 because this is not a private method :smile: Kindly put this method before line 9 :+1:
- [ ] As i mentioned above that your method should make all the data in descending order first and then it should get only 5 rows from data :blush: as shown below :arrow_down:

```ruby
  def recent_comments
    comments.order(created_at: :desc).limit(5)
  end
```